modules/cats-effect2/src/test/scala/pureconfig/module/catseffect2/CatsEffectSuite.scala

```diff
@@ -16,14 +16,18 @@ import pureconfig.{BaseSuite, ConfigReader, ConfigSource, ConfigWriter}
 final class CatsEffectSuite extends BaseSuite {
 
   case class SomeCaseClass(somefield: Int, anotherfield: String)
+  object SomeCaseClass {
+    def unapply(c: SomeCaseClass): Option[(Int, String)] = Some((c.somefield, c.anotherfield))
+  }
```

Comment:
As with the `CatsEffectSuite` of pureconfig-cats-effect, I think we can avoid this by defining the `ConfigWriter` differently.

Reply:
Done